pycbc/workflow/pegasus_files/local-site-template.xml

```diff
@@ -16,3 +16,4 @@
     <profile namespace="condor" key="use_x509userproxy">True</profile>
     <profile namespace="condor" key="+DESIRED_Sites">&quot;nogrid&quot;</profile>
     <profile namespace="condor" key="+IS_GLIDEIN">&quot;False&quot;</profile>
+    <profile namespace="condor" key="+WantIOProxy">&quot;True&quot;</profile>
```

Comment:
Nope, it's just going to kill the performance.  You don't need any changes to the job template at all -- just invoke `condor_chirp`.  See how CMS does it here:

https://github.com/cms-sw/cmssw/blob/master/FWCore/Services/plugins/CondorStatusUpdater.cc#L410